llvm/lib/Target/SPIRV/SPIRVTargetTransformInfo.cpp

```diff
@@ -0,0 +1,34 @@
+#include "SPIRVTargetTransformInfo.h"
+#include "llvm/IR/IntrinsicsSPIRV.h"
+
+namespace llvm {
```

Comment:
```suggestion
using namespace llvm;
```
https://llvm.org/docs/CodingStandards.html#use-namespace-qualifiers-to-implement-previously-declared-functions